opencti-platform/opencti-front/src/private/components/common/containers/ContainerStixCoreObjectPopover.jsx

```diff
@@ -231,8 +263,21 @@ class ContainerStixCoreObjectPopover extends Component {
     });
   }
 
+  closeReferencesPopup() {
+    this.setState({ referenceDialogOpened: false });
+  }
+
+  submitReference(values, { setSubmitting, resetForm }) {
+    const { displayRemove, displayDeleteMapping } = this.state;
+    const commitMessage = values.message;
+    const references = R.pluck('value', values.references || []);
```

Comment:
```suggestion
    const references = (values.references || []).map(ref => ref.value);
```
Trying to not use ramda